lib/handlers/slack.rb

```diff
@@ -31,8 +31,8 @@ def call
 
       begin
         client.chat_postMessage(channel: channel, text: message)
-      rescue ::Slack::Web::Api::Errors::TooManyRequestsError => e
-        raise(UserError, "TooManyRequestsError: #{e.message}")
+      rescue ::Slack::Web::Api::Errors => e
```

Comment:
Hmm yesy but:

> class TooManyRequestsError < ::Faraday::Error

https://github.com/slack-ruby/slack-ruby-client/blob/master/lib/slack/web/api/errors/too_many_requests_error.rb

Seems to be a special case...


```suggestion
      rescue ::Slack::Web::Api::Errors::TooManyRequestsError => e
        raise(UserError, "Unable to send Slack message. Slack replied with the following message: #{e.message}")
      rescue ::Slack::Web::Api::Errors => e
        raise(UserError, "Unable to send Slack message. Slack replied with the following message: #{e.message}")
```